data_structures/arrays/longest_consecutive_sequence.py

```diff
@@ -0,0 +1,63 @@
+"""
+author: Sarthak Sharma https://github.com/Sarthak950
+Website: https://sarthak950.netlify.app
+date:   4 OCT 2023
+Longest Consecutive Sequence Problem from LeetCode
+"""
+
+
+def longest_consecutive_sequence(nums: list[int]) -> int:
+    """
+    Finds the length of the longest consecutive sequence in a list of numbers.
+
+    Args:
+        nums (List[int]): A list of integers.
+
+    Returns:
+        int: The length of the longest consecutive sequence.
+
+    Examples:
+        >>> longest_consecutive_sequence([100, 4, 200, 1, 3, 2])
+        4
+        >>> longest_consecutive_sequence([1, 2, 3, 4, 5])
+        5
+        >>> longest_consecutive_sequence([5, 4, 3, 2, 1])
+        5
+        >>> longest_consecutive_sequence([])
+        0
+    """
```

Comment:
```suggestion
    """
    Finds the length of the longest consecutive sequence in a list of numbers.
    
    nums: A list of integers.
    int: The length of the longest consecutive sequence.

    >>> longest_consecutive_sequence([100, 4, 200, 1, 3, 2])
    4
    >>> longest_consecutive_sequence([1, 2, 3, 4, 5])
    5
    >>> longest_consecutive_sequence([5, 4, 3, 2, 1])
    5
    >>> longest_consecutive_sequence([])
    0
    """
```
Please read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#coding-style) to see how to indent docstrings.